spatial-history/index.md

```diff
@@ -1,39 +1,33 @@
-# This Map will eventually Become some kind of tour of Toronto.
+# The Montreal Video Game Cluster
+<h3>By Justin Capocci</h3>
+<h2 id=intro>Introduction</h2>
 
-Your first task will be to locate some real places.  You can get the precise locations at various services, e.g [latlong.net](http://www.latlong.net/).
-
-Then add those places to the marker array in [script.js](./script.js) (see instructions there).
-
-replace this text and the text with your essay. But be sure not to eliminate the html elements that
-are contained inside this textarea, or [your map won't work at all!][1] [1](#one)
-<figure>
-![](https://upload.wikimedia.org/wikipedia/commons/a/ab/Lolcat_in_folder.jpg "this is the alt text")<figcaption>caption</figcaption>
-</figure>
-It makes sense to put an introductory paragraph here, above the map and buttons. To change the bootswatch thenme, change "united" to one of the other theme names listed at [strapdownjs.com](http://strapdownjs.com/).
-
-[1]: http://hello.com
-[^1]: This is my first footnote
-
-<a id="#one">1: </a> some text here
+<p>In understanding the historical significance of spaces, clusters of economic activity are key to examining how places become associated culturally with specific industries. There are a number of well known clusters that have become a part of our collective consciousness, such as Wall Street being associated with investment banking, Silicon Valley with software development, and Hollywood with motion pictures. A less well known one that is nonetheless significant to the spatial history of the area is the video game development industry in Montreal, Quebec. Looking at the map below, it can be seen that the marked locations, each representing the large studios in Montreal (studios which employ over 200 employees), are within a very small area of only a few kilometres between its furthest points. The close proximity of these studios and the auxiliary companies that enable large scale game development are what establish the area as a cluster of development. By looking at the history of the cluster in the past three decades, the policies which produced it, its impact on the economic situation of Montreal, and its importance to the cultural image of the city as a centre for creativity and success in the gaming industry.</p>
```

Comment:
again, don't use direct html -- it stops you from taking advantage of markdown inside the tags.

Reply:
:heavy_check_mark:  this is precisely the kind of argument I'm looking for.